src/preferences/node/default-preference-server.ts

```diff
@@ -27,15 +27,28 @@ export interface PreferenceContribution {
 
 export class DefaultPreferenceServer implements IPreferenceServer {
 
+    defaultPrefs: Map<string, any>;
```

Comment:
please internal fields and methods `protected`, if they are constants then also `readonly`